assets/app/scripts/directives/oscKeyValues.js

```diff
@@ -148,6 +154,13 @@ angular.module("openshiftConsole")
       compile: function(element, attrs){
         if(!attrs.delimiter){attrs.delimiter = ":";}
         if(!attrs.keyTitle){attrs.keyTitle = "Name";}
+        if(!attrs.valueTitle){attrs.valueTitle = "Value";}
+        if(attrs.canBeEmpty === "true"){
+          attrs.canBeEmpty = true;
+        }else{
+          attrs.canBeEmpty = false;
+        }
```

Comment:
I'd probably flip it then to what you must have had: `preventEmpty`, then the default if not set is more obvious.